metamusic/clients/musicbrainz.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+"""
+Author : Salvador <salvador.estrella.ortiz@gmail.com>
+Date   : 2022-09-30
+Purpose: Query Metadata from MusicBrainz API
+"""
+
+import requests
+from time import sleep
+
+
+# --------------------------------------------------
+class MusicMetadata:
+    base_url = "http://musicbrainz.org/ws/2/"
+
+    @classmethod
+    def _pick_artist(cls, artist_list: list[dict]):
+        # TODO: Deal with artists that don't return a score equal to 100.
+        artist = None
+        for posible_artist in artist_list:
+            if not artist:
+                artist = posible_artist
+            elif artist["score"] < posible_artist["score"]:
+                artist = posible_artist
+        if artist and artist["score"] != 100:
+            raise NotImplementedError(
+                "Selection of artists with lower score than 100 not implemented yet"
+            )
+        return artist
+
+    @classmethod
+    def _artist_lookup(cls, artist_id: str) -> dict:
+        artist_lookup = f"{cls.base_url}artist/{artist_id}?inc=release-groups&fmt=json"
```

Comment:
**question**: What would be the suggested way to separate in multiple lines, from line 26 to 28 and line 33.

Reply:
For line 33, I would leave it as is, as it is a URL. Maybe use urljoin for joining cls.base_url, artist, and artist_id.

For line 27, you could do something like
```python
            raise NotImplementedError(
                "Selection of artists with lower"
                " score than 100 not implemented yet"
            )
```